# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build xz-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.69; am-1.15)
aclocal -I m4; autoheader; autoconf; automake -a -c
FORCE_UNSAFE_CONFIGURE=1 ./configure --libexecdir=/lib/tar --prefix=

# build
make

# install
install -d $DESTDIR/bin
install src/tar $DESTDIR/bin
install -d $DESTDIR/lib/tar
install rmt/rmt $DESTDIR/lib/tar

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/tar-build.pc << EOF
Name: tar
Version: 1.32
Description: GNU tape archiver
Requires: $REQUIRES

exec=\\
/bin/tar \\
/lib/tar/rmt
EOF

exit

# patch
--- tar-1.32.orig/configure.ac
+++ tar-1.32/configure.ac
@@ -23,6 +23,7 @@
 AC_CONFIG_HEADERS([config.h])
 AC_PREREQ([2.63])
 AM_INIT_AUTOMAKE([1.11 gnits tar-ustar dist-bzip2 dist-xz std-options silent-rules])
+AM_MAINTAINER_MODE
 
 # Enable silent rules by default:
 AM_SILENT_RULES([yes])

# source
https://ftp.gnu.org/gnu/tar/tar-1.32.tar.gz
